crates/sui-types/src/signature.rs

```diff
@@ -31,6 +48,7 @@ pub trait AuthenticatorTrait {
 /// This way MultiSig (and future Authenticators) can implement its own `verify`.
 #[enum_dispatch(AuthenticatorTrait)]
 #[derive(Debug, Clone, PartialEq, Eq, JsonSchema, Hash)]
+#[allow(clippy::large_enum_variant)]
```

Comment:
this doesn't look like it has any large variants? is this still needed?

Reply:
multisig is quite large and triggered this